resources/views/index.blade.php

```diff
@@ -79,7 +79,7 @@
     operationsSorter: {!! isset($operationsSorter) ? '"' . $operationsSorter . '"' : 'null' !!},
     configUrl: {!! isset($configUrl) ? '"' . $configUrl . '"' : 'null' !!},
     validatorUrl: {!! isset($validatorUrl) ? '"' . $validatorUrl . '"' : 'null' !!},
-    oauth2RedirectUrl: "{{ route('l5-swagger.'.$documentation.'.oauth2_callback', [], $useAbsolutePath) }}",
+    oauth2RedirectUrl: "{{ route('l5-swagger.'.$documentation.'.oauth2_callback', $useAbsolutePath) }}",
```

Comment:
These changes are reversing PR #411, I don't think you want to do that.

Reply:
Yes, this change was unintentional. please check it again